src/sources/demo_logs.rs

```diff
@@ -204,9 +213,18 @@ async fn demo_logs_source(
 
                     let events = events.into_iter().map(|mut event| {
                         let log = event.as_mut_log();
-
-                        log.try_insert(log_schema().source_type_key(), Bytes::from("demo_logs"));
-                        log.try_insert(log_schema().timestamp_key(), now);
+                        log_namespace.insert_vector_metadata(
+                            log,
+                            log_schema().source_type_key(),
+                            "source_type",
+                            "demo_logs",
+                        );
+                        log_namespace.insert_vector_metadata(
+                            log,
+                            log_schema().timestamp_key(),
+                            "ingest_timestamp",
```

Comment:
To clarify, are we moving to set `ingest_timestamp` on every source (regardless of the type of namespace)? That's what it looks like, just wanted to confirm.

Reply:
no, sources with the `Legacy` namespace will use `log_schema().timestamp_key()`, and the `Vector` namespace will use `ingest_timestamp`. These migrations shouldn't change anything when using the `Legacy` namespace.

Reply:
Bleh, I was still looking for where we switched between `legacy` and `vector` - forgetting the function takes both paths as args. Sorry late night confusion 😅